internal/state/state_test.go

```diff
@@ -815,40 +815,23 @@ func TestFourAddFourMinusOneGenesisValidators(t *testing.T) {
 	// we will keep the same quorum hash as to be able to add validators
 
 	// add 10 validators with the same voting power as the one added directly after genesis:
-	for i := 0; i < 10; i++ {
```

Comment:
I think we should keep previous test, and just add new one on top of it.

(Or maybe have a separate test func?)